libs/ledger/src/storage_unit/transaction_store_sync_service.cpp

```diff
@@ -234,6 +265,9 @@ TransactionStoreSyncService::State TransactionStoreSyncService::OnQuerySubtree()
 
     promise_id_to_roots_[promise.id()] = root;
     pending_subtree_.Add(root, promise);
+
+    subtree_requests_total_->increment();
+    ++requests_made;
```

Comment:
I'm putting comment on this line only because gihub review does not llow me to put it on unchanged code.
The comment is about the `FETCH_LOG_INFO(...)` which should be rather relocated to `OnResolvingSubtree()` method. Reason beying that we should print this when we actually received data for subtrees, as opposed to current situation where the print is done a bit prematurely when we send requests (thus it is missleading).